requirements.txt

```diff
@@ -1,3 +1,3 @@
 pylint-plugin-utils==0.7
-pylint==2.15.*
+pylint>=3, <4
```

Comment:
Please, pin the pylint version in order to avoid adding new lints and creating new CI red without an explicit validation from our side and advice to our team and community